tests/fixtures/config/env/test.yml

```diff
@@ -14,7 +14,7 @@ engines:
       type: path
       path: $SHOTGUN_TEST_ENGINE
 frameworks:
-  tk-framework-qtwidgets:
+  tk-framework-qtwidgets_v2.x.x:
```

Comment:
This is from the test that imports everything I mentionned in the PR description.